include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -116,21 +116,14 @@ class MMR92C04 {
         }
     }
 
-    auto get_pressure(mmr920C04::Registers reg, bool poll = false,
-                      uint16_t sample_rate = 0) -> void {
-        if (poll) {
-            poll_read(reg, sample_rate);
-        } else {
-            transact(reg);
-        }
+    auto get_pressure(mmr920C04::Registers reg) -> void {
+        uint32_t data = 0x0;
+        write(reg, data);
     }
 
-    auto get_temperature(bool poll = false, uint16_t sample_rate = 0) -> void {
-        if (poll) {
-            poll_read(mmr920C04::Registers::TEMPERATURE_READ, sample_rate);
-        } else {
-            transact(mmr920C04::Registers::TEMPERATURE_READ);
-        }
+    auto get_temperature() -> void {
+        uint32_t data = 0x0;
```

Comment:
Can you add some big TODOs here about implementing proper polling in a follow-up PR?